backend/src/entities/ClientEntity.ts

```diff
@@ -0,0 +1,25 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+// Creates an entity
+// Entity is a class that maps to a database table : https://orkhan.gitbook.io/typeorm/docs/entities
+
+@Entity()
```

Comment:
Maybe extend User? Like this: https://github.com/hack4impact-mcgill/MADA-v2/pull/38/files